packages/core/src/sass/account-wizard.scss

```diff
@@ -445,3 +445,230 @@
+                width: 90%;
+                white-space: break-spaces;
+            }
+        }
+    }
+
+    &__wrapper__question {
+        .dc-radio-group {
+            gap: 1.6rem;
+        }
+        @include mobile() {
+            .dc-radio-group__circle {
+                align-self: flex-start !important;
+            }
+        }
+    }
+
+    &__wrapper__dropdown {
+        gap: 2.4rem;
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+    }
+
+    .disable-pointer {
+        cursor: not-allowed;
+    }
+
+    @include mobile() {
+        .dc-dropdown__display-placeholder {
+            width: 100%;
+            .trading-assessment__form .dc-dropdown__display .dc-dropdown__display-placeholder-text-limit-fit {
+                width: 90%;
+            }
+        }
+
+        .dc-dropdown__display-placeholder--is-left-text.dc-dropdown__display-placeholder--is-title {
+            white-space: break-spaces !important;
+            line-height: 1.6rem;
+        }
+    }
+
+    &__existing_btn {
+        border-top: 2px solid var(--general-disabled);
+        display: flex;
+        justify-content: end;
+        padding: 1rem 2rem;
+        position: fixed;
+        bottom: 0;
+        width: 100%;
+    }
+
+    &__btn-group {
+        @include mobile() {
+            position: absolute;
+            bottom: 0;
+            padding: 1.6rem;
+            border-top: 2px solid var(--general-disabled);
+            width: 100%;
+            display: flex;
+            background: var(--general-main-1);
+
+            &--btn {
+                width: 100%;
+            }
+        }
+    }
+}
+
+.dc-mobile-dialog {
+    &__risk-acceptance {
+        margin-top: 9.4rem;
+        margin-inline: 1.6rem;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        row-gap: 2.4rem;
+    }
+    &__test-warning {
+        margin-top: 1.6rem;
+        .dc-modal-body {
+            padding-inline: 1.6rem;
+        }
+        .dc-modal-footer {
+            position: fixed;
+            bottom: 0;
+            padding: 1.6rem;
+            border-top: 2px solid var(--general-disabled);
+            width: 100%;
+            display: flex;
+            background: var(--general-main-1);
+            .dc-btn {
+                width: 100%;
+            }
+        }
+    }
+}
+
+.dc-modal__container_center-risk-modal {
+    .dc-modal-header {
+        border-bottom: 1px solid var(--general-section-1);
+    }
+    .dc-modal-body {
+        text-align: center;
+        .risk-acceptance__text {
+            margin-top: 2.4rem;
+        }
+        .verified-account__text {
+            margin-top: 2.4rem;
+        }
+    }
+
+    .dc-modal-footer {
+        justify-content: center;
+    }
+}
+
+.dc_modal__container_verified_account {
+    .dc-modal-body {
+        text-align: center;
+    }
+
+    .dc-modal-footer {
+        justify-content: center;
+    }
+}
```

Comment:
@likhith-deriv @aizad-deriv  please nest these styles as per the project sass nesting rules 🙂:
```.dc-modal__container {
    &_center-risk-modal {
          .dc-modal {
               &-header { ... }
               &-body { ... }
               &-footer { ... }
          }
       }
       &_verified_account {
           .dc-modal {
                &-body { ... }
               &-footer { ... }
            }
       }
  }```